steam/client/builtins/apps.py

```diff
@@ -244,3 +244,26 @@ def register_product_key(self, key):
             return EResult(resp.eresult), resp.purchase_result_details, details
         else:
             return EResult.Timeout, None, None
+
+    def request_free_license(self, app_ids):
+        """ Request license for free app
+        
+        :param app_ids: list of app ids
+        :type  app_ids: :class:`list`
+        :return: format ``(granted_appids, granted_packageids)``
+        :rtype: :class:`tuple`
+        
+        .. code:: python
+          [730] []
+        
+        """
+        resp = self.send_job_and_wait(MsgProto(EMsg.ClientRequestFreeLicense),
+                                      {'appids': map(int, app_ids)},
+                                      timeout=30,
```

Comment:
Can we lower the timeout to something more reasonable? Also, `resp` will be `None` on timeout and the code after will break